cmd/client/main.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func main() {
+	client := &http.Client{
+		CheckRedirect: func(req *http.Request, via []*http.Request) error {
+			return http.ErrUseLastResponse
+		},
+	}
+	scanner := bufio.NewScanner(os.Stdin)
+
+	const baseEndpoint = "http://localhost:8080/"
+	for {
```

Comment:
[CLARIFICATION] Please review this code. Please add more tests.

Reply:
I’ve moved the cmd/client/main.go code to the examples package. For now, I won’t be adding test coverage for this part because it’s more focused on demonstrating how to use the service